beep/conversion_schemas/structured_dtypes.yaml

```diff
@@ -0,0 +1,65 @@
+cycles_interpolated:
+  cycle_index: 'int32'
+  current: 'float32'
+  voltage: 'float32'
+  temperature: 'float32'
+  internal_resistance: 'float32'
+  charge_capacity: 'float64'
+  discharge_capacity: 'float64'
+  step_type: 'category'
+
+summary:
+  cycle_index: 'int32'
+  discharge_capacity: 'float64'
+  charge_capacity: 'float64'
+  discharge_energy: 'float64'
+  charge_energy: 'float64'
+  dc_internal_resistance: 'float32'
+  temperature_maximum: 'float32'
+  temperature_average: 'float32'
+  temperature_minimum: 'float32'
+  date_time_iso: 'object'
+  energy_efficiency: 'float64'
```

Comment:
Trying to understand the rationale for 64 bit precision for efficiency -- is it because degradation could be very sensitive to small changes in energy or coulombic efficiency?

Reply:
Yeah, I just thought that efficiency is something where the difference between 0.9999 and 0.999 is significant so adding more accuracy there might be helpful. If you think we can get away with float32, that would be better.